network/Cargo.toml

```diff
@@ -50,3 +57,8 @@ harness = false
 [[bench]]
 name = "network_bench"
 harness = false
+
+[features]
+default = []
+# fuzzing = ["proptest", "proptest_helpers", "tokio-codec"]
+fuzzing = ["proptest", "proptest_helpers"]
```

Comment:
:/ Hmm Have we finally made a call on what we're doing with these features? @huitseeker @metajack